packages/eslint-config-airbnb/README.md

```diff
@@ -12,7 +12,7 @@ We export three ESLint configurations for your usage.
 
 Our default export contains all of our ESLint rules, including ECMAScript 6+ and React. It requires `eslint`, `eslint-plugin-import`, `eslint-plugin-react`, and `eslint-plugin-jsx-a11y`. If you don't need React, see [eslint-config-airbnb-base](https://npmjs.com/eslint-config-airbnb-base).
 
-If you use yarn, run `yarn add --dev eslint-config-airbnb-base eslint-plugin-import eslint-plugin-react eslint-plugin-jsx-a11y`, and see below for npm instructions.
+If you use yarn, run `yarn add --dev eslint-config-airbnb eslint-plugin-import eslint-plugin-react eslint-plugin-jsx-a11y`, and see below for npm instructions.
```

Comment:
It should be sufficient but by running the `yarn add --dev eslint-config-airbnb` itself I got this error:

> Oops! Something went wrong! :(
> ESLint couldn't find the plugin "eslint-plugin-jsx-a11y". This can happen for a couple different reasons:
> 1. If ESLint is installed globally, then make sure eslint-plugin-jsx-a11y is also installed globally. A globally-installed ESLint cannot find a locally-installed plugin.
> 2. If ESLint is installed locally, then it's likely that the plugin isn't installed correctly. Try reinstalling by running the following:
>     npm i eslint-plugin-jsx-a11y@latest --save-dev
> If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.
> error Command failed with exit code 1.

My ESLint version is 3.19.0 (as not yet compatible with v4)
My Yarn version is v0.24.6

Adding all packages together also has a warning: "eslint-config-airbnb@15.0.1" has incorrect peer dependency "eslint-plugin-jsx-a11y@^5.0.1" and command fail with exit code 1.